### PR TITLE
Add notes about `tini`

### PR DESCRIPTION
**Pull request by yosifkit**
Clarify when tini is acceptable. Provide examples for "Clarity" section. Alternative flow for taking ownership of an official image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,14 @@ If you do not represent upstream and upstream becomes interested in maintaining
...
+When taking over an existing repository, please ensure that the entire Git history of the original repository is kept in the new upstream-maintained repository to make sure the review process isn't stalled during the transition. This is most easily accomplished by forking the new from the existing repository, but can also be accomplished by fetching the commits directly from the original and pushing them into the new repo (ie, `git fetch https://github.com/jsmith/example.git master`, `git rebase FETCH_HEAD`, `git push -f`). On github, an alternative is to move ownership of the git repository. This can be accomplished without giving either group admin access to the other owner's repository:
+
+-	create temporary intermediary organization
+-	give old and new owners admin access to intermediary organization
+-	old owner transfers repo ownership to intermediary organization
+-	new owner transfers repo ownership to its new home
+	-	recommend that old owner does not fork new repo back into the old organization to ensure that github redirects will just work
```
Comment (tianon) [inline README.md]: Do you want to mention that we have the https://github.com/docker-library-transitioner organization registered for this express purpose, and that we're happy to help use it to connect parties for repository transfer?

**Comment by yosifkit** [comment]
Here is your documentation @psftw re: https://github.com/docker-library/official-images/pull/927#issuecomment-127258395 :smile:

**Comment by psftw** [comment]
:+1: Thanks, this is great!

File: README.md [hunk 2]
```diff
@@ -82,6 +89,28 @@ All official images should provide a consistent interface. A beginning user shou
...
+-	[ruby:2.2](https://github.com/docker-library/ruby/blob/e34b201a0f0b49818fc8373f6a9148e13d546bdf/2.2/Dockerfile)
+
+#### init
+
+Following the docker guidelines it is highly recommended that the resulting image be just one concern per container; predominantly this means just one process per container, so there is no need for a full init system. There are two situations where an init-like process would be helpful for the container. The first being signal handling. If the process launched does not handle `SIGTERM` by exiting, it will not be killed since it is PID 1 in the container (see "NOTE" at the end of the [Foreground section](https://docs.docker.com/reference/run/#foreground) in the docker docs). The second situation would be zombie reaping. If the process spawns child processes and does not properly reap them it will lead to a full process table, which can prevent the whole system from spawning any new processes. For both of these concerns we recommend [tini](https://github.com/krallin/tini). It is incredibly small, has minimal external dependencies, fills each of these roles, and does only the necessary parts of reaping and signal forwarding.
```
Comment (md5) [inline README.md]: Shouldn't this be "Docker guidelines" instead of "docker guidelines"?

Comment (yosifkit) [inline README.md]: fixed :+1:

**Comment by tianon** [comment]
LGTM

**Pull request merged**